src/Tribe/Repository.php

```diff
@@ -1794,7 +1794,9 @@ protected function apply_default_modifier( $key, $value ) {
 				$args = [ 'p' => $value ];
 				break;
 			case 'search':
-				$args = [ 's' => $value ];
+				if ( ! empty( $value ) ) {
```

Comment:
I would make the comparison stricter. String zero, (`"0"`) will evaluate to empty while still being a legit, albeit strange, search term.
```suggestion
				if ( '' !== $value ) ) { 
```